README.md

```diff
@@ -0,0 +1,24 @@
+[![oomph-lib](doc/figures/oomph_logo.png)](http://oomph-lib.maths.man.ac.uk)
+
+[![License: LGPL v2.1](https://img.shields.io/badge/License-LGPL%20v2.1-blue.svg)](https://www.gnu.org/licenses/lgpl-2.1)
+[![Ubuntu tests status](https://github.com/oomph-lib/oomph-lib/actions/workflows/self-tests-ubuntu.yaml/badge.svg)](https://github.com/oomph-lib/oomph-lib/actions/workflows/self-tests-ubuntu.yaml/badge.svg)
```

Comment:
Ahh nice, thanks for adding badges! I now see why you wanted separate workflows, would have been useful to explain this in the PR body, it seemed like a strange choice initially.

Reply:
Good idea! I've updated the PR text accordingly.